### PR TITLE
doc: improve merge labels by introducing merge-master-only and merge-patch-only labels

### PR DESCRIPTION
**Pull request by IgorMinar**

### CODE REVIEW

**Comment by IgorMinar** [comment]
Once this is approved, I'll go ahead an make the actual changes to tags on github.

File: docs/TRIAGE_AND_LABELS.md [hunk 1]
```diff
@@ -108,16 +108,29 @@ closing or reviewing PRs is a top priority ahead of other ongoing work.
...
 
-* `pr_state: WIP` - PR is experimental or rapidly changing. Not ready for review or triage.
-* `pr_state: blocked` - PR is blocked on an issue or other PR. Not ready for review or triage.
+* `PR state: WIP` - PR is experimental or rapidly changing. Not ready for review or triage.
+* `PR state: blocked` - PR is blocked on an issue or other PR. Not ready for review or triage.
+
+
+## PR Target
+
+In our git workflow we merge changes either to the `master` branch, the most recent patch branch (e.g. `4.3.x`), or both.
```
Comment (vicb) [inline docs/TRIAGE_AND_LABELS.md]: not sure but should it be to either a, b, or both or either to a, to b, or to both ?

**Review by mhevery — APPROVED** [review]

**Comment by mhevery** [comment]
Should we add information as to who is responsible for adding these labels?

**Review by alxhub — APPROVED** [review]

File: docs/TRIAGE_AND_LABELS.md (hunk 1 above)
Comment (IgorMinar) [inline docs/TRIAGE_AND_LABELS.md]: changed to "to both"

**Pull request merged**